README.rst

```diff
@@ -32,6 +32,17 @@ Key Features
 - Proper rate limit handling
 - Optimised in both speed and memory
 
+Why choose Nextcord?
+--------------------
+
+- No breaking changes unless necessary
```

Comment:
Maybe add welcome screen?

Reply:
Something like "Here's a list of reasons why you should start using Nextcord instead of other alternatives"? Because I don't quite understand what else a "welcome screen" would be.

Reply:
Welcome screen support. You know those channels when you join a server?

Reply:
see pr #66 for what they're on about